Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.cpp

```diff
@@ -57,13 +60,105 @@ namespace std
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = s_duplicateVerticesTitle;
+            actionProperties.m_description = s_duplicateVerticesDesc;
+            actionProperties.m_category = "Vertex Selection";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                []
+                {
+                    EditorVertexSelectionRequestBus::Broadcast(&EditorVertexSelectionRequests::DuplicateSelectedVertices);
+                }
+            );
+
+            hotKeyManagerInterface->SetActionHotKey(actionIdentifier, "Ctrl+D");
+        }
+
+        // Delete Vertices
+        {
+            constexpr AZStd::string_view actionIdentifier = "o3de.action.vertexSelection.delete";
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = s_deleteVerticesTitle;
+            actionProperties.m_description = s_deleteVerticesDesc;
+            actionProperties.m_category = "Vertex Selection";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                []
+                {
+                    EditorVertexSelectionRequestBus::Broadcast(&EditorVertexSelectionRequests::DeleteSelectedVertices);
+                }
+            );
+
+            hotKeyManagerInterface->SetActionHotKey(actionIdentifier, "Delete");
+        }
+
+        // Clear Vertex Selection
+        {
+            constexpr AZStd::string_view actionIdentifier = "o3de.action.vertexSelection.clearSelection";
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = s_deselectVerticesTitle;
+            actionProperties.m_description = s_deselectVerticesDesc;
+            actionProperties.m_category = "Vertex Selection";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                []
+                {
+                    EditorVertexSelectionRequestBus::Broadcast(&EditorVertexSelectionRequests::ClearVertexSelection);
+                }
+            );
+
+            hotKeyManagerInterface->SetActionHotKey(actionIdentifier, "Esc");
+        }
+    }
+
+    void EditorVertexSelection::BindEditorVertexSelectionActionsToMenus()
+    {
+        auto menuManagerInterface = AZ::Interface<AzToolsFramework::MenuManagerInterface>::Get();
+        AZ_Assert(
+            menuManagerInterface,
+            "EditorVertexSelection - could not get MenuManagerInterface on BindActionsToMenus.");
+
+        menuManagerInterface->AddActionToMenu(EditMenuIdentifier, "o3de.action.vertexSelection.duplicate", 6000);
```

Comment:
suggestion: Might be good to move these (`"o3de.action.vertexSelection.duplicate"`) to constants at the top of the file (so they aren't repeated here and in `RegisterEditorVertexSelectionActions`)